docker-compose.yml

```diff
@@ -14,9 +14,15 @@ services:
     depends_on:
       - postgres
 
+  cli5:
+    image: conjurinc/cli5
+    environment:
+      CONJUR_APPLIANCE_URL: http://conjur
+      CONJUR_ACCOUNT: cucumber
+
   cuke-master:
     container_name: api-go-conjur-v4
-    image: registry.tld/conjur-appliance-cuke-master:4.9-stable
+    image: registry2.itci.conjur.net/conjur-appliance-cuke-master:4.9-stable
```

Comment:
i agree with @dustinmm80 on this https://github.com/cyberark/conjur/pull/221#discussion_r132534230. there are several good reasons to keep it as `registry.tld` including consistency.